content/blog/gradle-tutorial/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "FrontLine with Gradle Tutorial"
+description: "step by step setup of FrontLine"
+lead: ""
+date: 2021-03-19T17:03:31+01:00
+lastmod: 2021-03-19T17:03:31+01:00
+draft: false
+images: []
+weight: 50
+images: []
```

Comment:
duplicate

Reply:
I need to add this conf in each file

Reply:
`images` entry is duplicate in the same file